modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,39 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0].
+
+[id="logging-release-notes-5-8-0-enhancements"]
+== Enhancements
+* With this update, you can have finer control over access to workload logs in the LokiStack. (link:https://issues.redhat.com/browse/LOG-3841[LOG-3841])
```

Comment:
Suggestion for LOG-3841 - 'With this update, LokiStack administrator can have more fine grained control over who accesses what logs by granting access to logs on a namespace basis.'